### ISSUE TITLE
Install public version of spartacus

### ISSUE THREAD

**Issue by hackergil** [SAP/cloud-commerce-spartacus-storefront#166]
we need a public spartacus to do real testing on mobile. we didn't want to rely on deploying spartacus thru modelt.

**Comment by hackergil** [comment]
We need someone with admin access to enable github-pages for our repo

**Comment by hackergil** [comment]
Pull request #181

**Comment by hackergil** [comment]
Blocked by #291

**Comment by hackergil** [comment]
Pull request #319

**Comment by hackergil** [comment]
Public version of Spartacus courtesy of github pages. You can check it out here:

https://sap.github.io/cloud-commerce-spartacus-storefront/

It will be updated every time we do a merge to develop
It uses our public model t as a backend (the same one that we use for e2e tests)

**Comment by gladius-mtl** [comment]
Created a doc follow-up ticket: GH-496. Moving this ticket to QA.

**Comment by Xymmer** [comment]
created bug #548 
Doesn't work on chrome for some people
